.arbitrum.mainnet.json

```
@@ -0,0 +1,84 @@
+{
+  "proxyAdmin": "",
+  "nativeRelayer": "",
+  "timelock": "",
+  "opMultiSig": "",
+  "usdPlaceholder": "",
+  "tokens": {
+    "weth": ""
```

no need to remove weth token
```suggestion
    "weth": ""
```